configure

```diff
@@ -2974,8 +2997,18 @@ POSTGRES_BUILDDIR="$POSTGRES_BUILDDIR"
 
 ac_config_files="$ac_config_files Makefile.global"
 
-ac_config_headers="$ac_config_headers src/include/citus_config.h"
+ac_config_headers="$ac_config_headers src/include/citus_config.h src/include/citus_version.h"
+
+
 
+cat >>confdefs.h <<_ACEOF
+#define CITUS_BINARY_VERSION "$citus_binary_version"
```

Comment:
Get rid of `CITUS_BINARY_VERSION`. We'll have `CITUS_VERSION` (`6.1.0`), `CITUS_MAJORVERSION` (`6.1`), and `CITUS_EXTENSIONVERSION` (`6.1-10`).